### ISSUE TITLE
Slow text unexpectedly dismissed when what_args["slow_abortable"] is False

### ISSUE THREAD

**Issue by dansalvato** [renpy/renpy#1137]
Not sure if this is intended behavior, so I apologize if it is.

If a character has slow text but slow_abortable is False, then clicking will cause the dismiss event and advance to the next line of script, even while the slow text is not finished being displayed.

It seems like `what_args["slow_abortable"] = False` should suppress the dismiss event until the text is finished displaying.

Here is a proposed change in `text/text.py`.

Current:
```python
    def event(self, ev, x, y, st):
        """
        Space, Enter, or Click ends slow, if it's enabled.
        """

        if self.slow and renpy.display.behavior.map_event(ev, "dismiss") and self.style.slow_abortable:
            self.call_slow_done(st)
            self.slow = False
            raise renpy.display.core.IgnoreEvent()
```

New:
```python
    def event(self, ev, x, y, st):
        """
        Space, Enter, or Click ends slow, if it's enabled.
        """

        if self.slow and renpy.display.behavior.map_event(ev, "dismiss"):
            if self.style.slow_abortable:
                self.call_slow_done(st)
                self.slow = False
            raise renpy.display.core.IgnoreEvent()
```

**Comment by renpytom** [comment]
This is intended behavior. 

We don't want to get in the case where we simply ignore a player's click without taking any action, since that means the game has become unresponsive. So we take the action we would have taken anyway, which is to dismiss the dialogue line.

**Comment by dansalvato** [comment]
Yeah, I figured that slow_abortable was probably an old relic that you didn't expect to be manually used nowadays. Given your philosophy on avoiding unskippable game content (transitions, renpy.pause, etc.), it makes sense for Ren'Py to not allow for unskippable slow text. I just had a very unique use case, but I already came up with a workaround that achieves the same thing, although it's a bit hackier.
